rotkehlchen/externalapis/beaconchain/service.py

```diff
@@ -250,14 +250,28 @@ def get_validator_data(
 
     def get_and_store_produced_blocks(
             self,
-            indices_or_pubkeys: list[int | Eth2PubKey],
+            indices: list[int],
     ) -> None:
         with self.produced_blocks_lock:
-            return self._get_and_store_produced_blocks(indices_or_pubkeys)
+            # Get indices for validators that are either active, exited but never queried, or
+            # exited and queried but exited timestamp is after last query timestamp.
+            with self.db.conn.read_ctx() as cursor:
+                placeholders = ', '.join(['?'] * len(indices))
+                key_name = DBCacheDynamic.LAST_PRODUCED_BLOCKS_QUERY_TS.value[0].replace('{index}', '')  # noqa: E501
```

Comment:
isn't this a constant value? Essentially `last_produced_blocks_query_ts_` ? Why not just do something like `DBCacheDynamic.LAST_PRODUCED_BLOCKS_QUERY_TS[:30]` and then you get `last_produced_blocks_query_ts_` without any other extra logic?